generator_pass_flags/1.in

```diff
@@ -0,0 +1,3 @@
+1
```

Comment:
I general, I don't like this legacy generator interface. It should never have been created and `generator_args` is just better.